app/uk/gov/hmrc/agentclientrelationshipsfrontend/config/Constants.scala

```diff
@@ -36,7 +36,7 @@ object Constants {
   val ClientTypeFieldName = "clientType"
   val ClientServiceFieldName = "clientService"
   val ClientNameFieldName = "clientName"
-  val ConfirmationFieldName = "accepted"
+  val ClientConfirmationFieldName = "confirm-client"
```

Comment:
Just for consistency could you change this and the message key prefixes to confirmClient